mod-orders/schemas/patch_pieces_request.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Patch pieces request",
+  "type": "object",
+  "properties": {
+    "pieces": {
+      "description": "An array of piece record IDs",
+      "type": "array",
+      "items": {
+        "description": "UUID of an piece record",
+        "$ref": "../../common/schemas/uuid.json"
+      },
+      "uniqueItems": true
+    },
+    "receivingStatus": {
+      "description": "The status to be applied to the specified pieces",
+      "type": "string",
+      "enum": [
+        "Received",
+        "Expected",
+        "Late",
+        "Claim delayed",
+        "Claim sent",
+        "Unreceivable"
+      ]
+    },
+    "claimingInterval": {
+      "description": "Time interval (in days) for processing claims related to this piece",
+      "type": ["integer", "null"]
```

Comment:
why we need ["integer", "null"] instead of just integer, do we have the same in other places?